packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -83,6 +83,7 @@ The following table provides a list of all available email templates in an Okta
 | LDAP User Activation | Welcome to Okta! | `LDAPUserActivation` | Sent to your new LDAP users who must follow the provided link to complete the Okta sign-up process |
 | Send Push Verify Activation Link | Push Verify Activation Email | `OktaVerifyActivation` | Sent to users who must follow the provided link to download Okta Verify Push for multifactor authentication on their mobile device |
 | Registration - Activation | Activate Account | `RegistrationActivation` | Sent to users who must follow the provided link to complete their self-registration |
+| Registration - Activation (Self-service Registration) | Activate Account | `SelfServiceRegistrationActivation` | Sent to users who can follow the provided link or use an OTP to complete their self-registration |
```

Comment:
Confirmed with design the subject line should be: ```Welcome to (`instanceDisplayName`)!```

Reply:
@brentschaus-okta I spoke with Bhagya the PM for this feature, and determined we don't need to add a new row. This email template is actually the same as the one above it for `Registration -Activation` it just replaces the link with a much better link experience (magic link) if user has the EA FF turned on, so there's nothing for a dev to do

might want to look at https://developer.okta.com/docs/guides/oie-embedded-sdk-use-case-self-reg/java/main/#_2-confirm-that-the-org-application-is-assigned-to-everyone instead to see if we should add an EA feature to that guide